CorsixTH/Lua/entities/humanoids/staff/handyman.lua

```diff
@@ -38,6 +38,7 @@ end
 function Handyman:die()
   self:unassignTask()
   Staff.die(self)
+  self.hospital:handymanDeath(self)
```

Comment:
onHandymanDeath might be nicer. But I note in hospital the humanoid one doesn't follow that trend either... Maybe it can stay.